src/kv/kv_types.h

```diff
@@ -680,6 +684,8 @@ namespace kv
     // Thus, a large rollback is one which did not result from the map creating
     // issue. https://github.com/microsoft/CCF/issues/2799
     virtual bool should_rollback_to_last_committed() = 0;
+
+    bool force_ledger_chunk = false;
```

Comment:
@jumaffre I hate having to modify `AbstractChangeSet`, but somehow I need this one bit of information to be carried from `ds->apply()` to `ledger->put_entry(... ds->force_ledger_chunk ... )` in `raft.h`. The only alternative I can think of is to add a flag to `aft::Store`, but that's not much of a difference. Do you have a better idea?

Reply:
As discussed, this will go anyway once https://github.com/microsoft/CCF/issues/3613 is implemented.

Reply:
As discussed, this will go anyway once https://github.com/microsoft/CCF/issues/3613 is implemented.